target/dovecot/90-quota.conf

```diff
@@ -42,8 +42,8 @@ service quota-warning {
 service quota-status {
     executable = quota-status -p postfix
     inet_listener {
-    port = 65265
-    # You can choose any port you want
+        address = localhost
```

Comment:
Could localist here potential resolve to an IOv6 address? Would it be better to use `127.0.0.1`?

Reply:
if the docker-container is set up properly, that shouldn't be the case.

but i also don't see any reason against changing it, so i will.

Reply:
well, technically, other components are using localhost to connect. so unless any of them have a non-standard way of resolving localhost or prefers ipv4 over ipv6 (or vice-versa), using `localhost`, dovecot will use the same as the other applications.

then again, the day anyone starts turning of ipv4 addresses for the loopback interface is probably far in the future.